docs/intro.rst

```diff
@@ -3,15 +3,27 @@
 Introduction
 ============
 
+.. pull-quote::
+
+    The relational model is still the most effective method
+    of representing data. And SQL as a declarative,
```

Comment:
A reference to SQL out of context feels a bit weird here, let's drop the quote for now and find better phrasing.

Reply:
OK